### PR TITLE
Use better label for system browser

### PR DESCRIPTION
**Pull request by jeluard**
### Summary:

Update label used when opening a system browser. It will now reflect the current platform.

status: ready

### CODE REVIEW

File: src/status_im/ui/components/list_selection.cljs [hunk 1]
```diff
@@ -30,17 +30,20 @@
          :options     (message-options message-id old-message-id text)
          :cancel-text (i18n/label :t/message-options-cancel)}))
 
+(defn- platform-web-browser []
+  (if platform/ios? :t/browsing-open-in-ios-web-browser :t/browsing-open-in-android-web-browser))
```
Comment (cammellos) [inline src/status_im/ui/components/list_selection.cljs:34]: no desktop?

**Review by cammellos — APPROVED** [review]

File: src/status_im/ui/components/list_selection.cljs (hunk 1 above)
Comment (jeluard) [reply src/status_im/ui/components/list_selection.cljs:34]: No browser on desktop for now? And not sure there would be a distinction between browsers as on mobile. But good point.

**Review by goranjovic — APPROVED** [review]

**Comment by churik** [comment]
@jeluard 
Tested on IOS and Android in 1-1, public and group chats.
Noticed that for all languages except English labels are the same.
Will they be translated separately?

**Comment by jeluard** [comment]
@churik Yes the old label is used for both platforms for most languages. No plan to translate them for now.

**Pull request merged**